seccomp not found during build process

After running make dep && make lint successfully, I get the following error during the build process on my CentOS Linux release 7.6.1810 (Core)  system with libseccomp-devel.x86_64             2.3.1-3.el7 installed.  PKG_CONFIG_PATH is set to /usr/lib/pkgconfig where the libseccomp.pc is installed.   Please provide guidance as to how I might resolve the issue.

$ make
 GO bin/sycri
 WARNING: seccomp is not found, ignoring
# github.com/seccomp/libseccomp-golang
../../../../pkg/mod/github.com/seccomp/libseccomp-golang@v0.9.0/seccomp_internal.go:23:2: error: #error Minimum supported version of Libseccomp is v2.1.0
 #error Minimum supported version of Libseccomp is v2.1.0
  ^
make: *** [bin/sycri] Error 2

--- a/Makefile
+++ b/Makefile
@@ -14,7 +14,7 @@ CRI_CONFIG_INSTALL := /usr/local/etc/sycri/sycri.yaml
 
 all: $(SY_CRI) $(FAKE_SH)
 
-$(SY_CRI): SECCOMP := "$(shell echo '#include <seccomp.h>\nint main() { }' | gcc -x c -o /dev/null -lseccomp - >/dev/null 2>&1; echo $$?)"
+$(SY_CRI): SECCOMP := "$(shell printf "#include <seccomp.h>\nint main() { seccomp_syscall_resolve_name(\"read\"); }" | gcc -x c -o /dev/null - -lseccomp >/dev/null 2>&1; echo $$?)"
 $(SY_CRI): export GO111MODULE=on
 $(SY_CRI):
 	@echo " GO" $@

